lib/rules/require-newline-before-block-statements.js

```diff
@@ -97,6 +97,7 @@ module.exports.prototype = {
             errors.assert.differentLine({
                 token: prevToken,
                 nextToken: openingBrace,
+                allowExtra: true,
```

Comment:
Is there a test you can think of that I'm missing? require-newline-block-statements had tests for both and I made sure those worked. I also have tests for token-assert differentLine that tests allowing extra, as well as requiring strict. :-)